### PR TITLE
Mm 46094 long run name issue

### PR DESCRIPTION
**Pull request by gbochora**
#### Summary
This PR fixes run long name truncate bug in the RDP. 

https://user-images.githubusercontent.com/4368372/185310622-b13062ee-9f36-4209-aede-e14ab8236fa7.mp4

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46094

#### Checklist

~~- [ ] Telemetry updated~~
~~- [ ] Gated by experimental feature flag~~
- [ ] Unit tests updated

### CODE REVIEW

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx [hunk 1]
```diff
@@ -312,7 +313,7 @@ const Main = styled.main`
 const Body = styled(RowContainer)`
 `;
 
-const Header = styled.header`
+const Header = styled.header`
```
Comment (trilopin) [inline webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:316]: nit: remove extra trailing space

**Review by trilopin — APPROVED** [review]
it works perfectly!

File: webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx [hunk 2]
```diff
@@ -286,7 +286,7 @@ const ColumnContainer = styled.div`
 const Container = styled(ColumnContainer)`
```
Comment (trilopin) [inline webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx:286]: maybe we should make this style.div

**Review by calebroseland — APPROVED** [review]
Excellent!

**Pull request merged**